src/components/SketchParam/index.js

```diff
@@ -1,16 +1,26 @@
 import React from 'react'
+import styled from 'styled-components'
 import PropTypes from 'prop-types'
-import Collapsible from '../../containers/Collapsible'
+import Revealer from '../../components/Revealer'
 import Param from '../../containers/Param'
 import Range from '../../containers/Range'
 import InputLinkUI from '../../containers/InputLinkUI'
+import theme from '../../utils/theme'
+
+const Top = styled.div`
+  margin-bottom: 0.5rem;
+  padding-bottom: 0.5rem;
+  border-bottom: ${theme.lineColor1} 1px dashed;
+`
 
 const SketchParam = ({ nodeId, sketchId }) => (
   <Param nodeId={nodeId} sketchId={sketchId}>
-    <Collapsible {... { title:'Advanced' }}>
+    <Top>
+      <InputLinkUI nodeId={nodeId} />
+    </Top>
+    <Revealer title='Advanced' isOpen>
```

Comment:
I generally don't like using little libraries such as the one you implemented. It's quick and easy to implement but then when you want to do something a bit more custom, things get hacky. I made a little component called `Revealer` that does the same thing. :)